rule_filter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rule_filter
 description: TMail rule_filter extension
-publish_to: 'none'
+publish_to: "none"
```

Comment:
please revert it

Reply:
Done

Reply:
Looks like you forgot to revert it. @tddang-linagora